File: squad-bsa_Manar-Adel_Cprogramming-Part3/squad-bsa_Manar-Adel_GuessingGame.c
#include<stdio.h>

//Guessing Game

int main(){

    int secretNumber=5;
    int guess;
    int guessCount=0;
    int guessLimit=3;
    int outOfGuesses=0;



    while(guess!=secretNumber && outOfGuesses==0){
        if(guessCount<guessLimit){
        printf("Enter a number: ");
        scanf("%d",&guess);
        guessCount++;
        }
        else{
            outOfGuesses=1; 
        }
    }

    if(outOfGuesses){
        printf("Out of Guessess");
    }
    else{
        printf("YOU WON!");
    }


}

File: squad-bsa_Manar-Adel_Python-Part3/squad-bsa_Manar-Adel_GuessingGame.py
#Guessing Game

secret_word="giraffe"
guess=""
guess_count=0
guess_limit=3
out_of_guesses=False

while guess!=secret_word and not(out_of_guesses):
    if guess_count<guess_limit:
        guess=input("Enter Guess: ")
        guess_count+=1
        if guess != secret_word:
            print("You have "+str(guess_limit-guess_count)+" guesses left")

    else:
        out_of_guesses=True   

if out_of_guesses:
    print("OUT OF GUESSES, YOU LOSE")
else:    
    print("CORRECT")    